Documentation/source/index.rst

```diff
@@ -83,6 +83,7 @@ Inputs
   objects/ColorWell
   objects/ComboBox
   objects/PopUpButton
+  objects/ActionButton
   objects/RadioGroup
   objects/Slider
 
```

Comment:
not included in the index: `VanillaMenuBuilder` – should it also be listed here?